# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: dom/l10n/tests/mochitest/dom_localization/test_overlay_missing_all.html
# Commit: 85194e5b7a9e
# Full Hash: 85194e5b7a9e6ed9114ee46aef0b5efb819cca9b
# Author: Zibi Braniecki <zbraniecki@mozilla.com>
# Date: 2021-08-04 03:56:33
# Regressor Bug: 1613705
# File Overlap Count: 1
# Description:
#   Bug 1613705 - [localization] part9: Update DOMLocalization to match Localization constructor. r=nika,emilio
#   
#   Depends on D113691
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D113692
# ==============================================================================

diff -r 181487bdf6c0 -r 85194e5b7a9e dom/l10n/tests/mochitest/dom_localization/test_overlay_missing_all.html
--- a/dom/l10n/tests/mochitest/dom_localization/test_overlay_missing_all.html	Tue Aug 03 16:25:13 2021 +0000
+++ b/dom/l10n/tests/mochitest/dom_localization/test_overlay_missing_all.html	Tue Aug 03 16:25:13 2021 +0000
@@ -7,18 +7,15 @@
   <link rel="stylesheet" type="text/css" href="chrome://mochikit/content/tests/SimpleTest/test.css">
   <script type="application/javascript">
   "use strict";
-  async function* generateBundles(resourceIds) {
-    const bundle = new FluentBundle("en-US");
-    // No translations!
-    yield bundle;
-  }
+  const l10nReg = new L10nRegistry();
 
   SimpleTest.waitForExplicitFinish();
   addLoadEvent(async () => {
     const domLoc = new DOMLocalization(
       [],
       false,
-      { generateBundles },
+      l10nReg,
+      ["en-US"],
     );
 
     await domLoc.translateFragment(document.body).then(() => {